tasks/src/feature-extraction/about.md

```diff
@@ -0,0 +1,30 @@
+## Use Cases
+
+One everyday use case for feature extraction is in natural language processing (NLP) tasks, where the goal is to process and analyze large amounts of text data. For example, a machine learning model might be trained to identify the sentiment of a piece of text (e.g., whether it is positive, negative, or neutral). In order to do this, the model needs to be able to understand the words and phrases in the text, as well as the overall context and meaning.
+To enable the model to understand the text, we need to extract features from the data that are relevant to the task at hand. This might involve converting the text into numerical vectors, where each element of the vector represents a particular word or phrase in the text. These vectors can then be fed into the machine learning model, which can use them to learn patterns and make predictions about the sentiment of new text.
+Overall, feature extraction is an essential step in many machine learning tasks, as it helps to reduce noise and irrelevant information in the data and make it more usable by the algorithms. By extracting relevant features from the data, we can improve the model's performance and make it better at solving the task at hand.
+ 
+## Task Variants
+
+Feature extraction is extracting relevant information from a dataset and converting it into a set of features that can be used to train a machine learning model. There are many different approaches to feature extraction, and the specific technique best suited for a given problem will depend on the data's characteristics and the analysis's goals. Some common approaches to feature extraction include:
```

Comment:
Most of the methods below are for feature engineering/finding important features to either remove less important features or to scale the features such that model would converge, while feature extraction is about creating a new feature subspace, e.g. word embeddings. Dimensionality reduction might count as you still derive a new set of feature subspace but rest is more about feature engineering IMHO.

Reply:
Ahh, I thought feature engineering is a subset of feature extraction since we are listing out the important features. I guess I might have to research more. Will do it and update you asap.

Reply:
@merveenoyan I have added the different techniques do let me know if it's fine or not, thanks for helping me :)